packages/ipfs/src/core/ipns/routing/pubsub-datastore.js

```diff
@@ -10,8 +10,9 @@ const withIs = require('class-is')
 
 const errcode = require('err-code')
 const debug = require('debug')
-const log = debug('ipfs:ipns:pubsub')
-log.error = debug('ipfs:ipns:pubsub:error')
+const log = Object.assign(debug('ipfs:ipns:pubsub'), {
+  error: debug('ipfs:ipns:pubsub:error')
+})
 
 // Pubsub datastore aims to manage the pubsub subscriptions for IPNS
 class IpnsPubsubDatastore {
```

Comment:
```suggestion
module.exports = class IpnsPubsubDatastore {
```

Reply:
or maybe we can remove `class-is` dependency completely

Reply:
@Xmader could you please elaborate what are you suggesting here ? I would rather keep this diff as small as possible, and proposed change does not appears strictly necessary, in which case I would much rather defer that to followups.

Reply:
`class-is` dependency doesn't have proper TypeScript defs, so this will result an `any` type in the generated `.d.ts` file

Reply:
This can be done in the followups

Reply:
Thanks for the explanation. I think we should just rip out the class-is. No one seems to like it and just tends to get in the way.